pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright Red Hat
+  ~ 
+  ~ SPDX-License-Identifier: LGPL-2.1-only
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.narayana.checkstyle</groupId>
```

Comment:
I would suggest to use only `io.narayana`, the fact it's a checkstyle artifact is defined in `artifactId`